iothub/service/src/Registry2/DevicesClient.cs

```diff
@@ -31,7 +31,7 @@ public class DevicesClient
         private const string JobsListUriFormat = "/jobs";
         private const string JobsCreateUriFormat = "/jobs/create";
         private const string DeviceStatisticsUriFormat = "/statistics/devices";
-        private const string ServiceStatisticsUriFormat = "/statistics/service?" + ClientApiVersionHelper.ApiVersionQueryString;
```

Comment:
This service API string is added to every HTTP request by the HTTP client, so there is no need to add it to any request URI format variable